airflow-core/src/airflow/models/deadline.py

```diff
@@ -155,7 +155,7 @@ def evaluate_with(self, **kwargs: Any) -> datetime:
             if extra_kwargs := kwargs.keys() - filtered_kwargs.keys():
                 self.log.debug("Ignoring unexpected parameters: %s", ", ".join(extra_kwargs))
 
-            return self._evaluate_with(**filtered_kwargs)
+            return self._evaluate_with(**filtered_kwargs) + interval
```

Comment:
Why does this need to change?

Reply:
That is what it was supposed to be returning.   The `deadline` table should be storing the calculated deadline time, which is the reference plus the interval